src/test/java/org/folio/oaipmh/service/impl/SetServiceImplTest.java

```diff
@@ -1,30 +1,56 @@
 package org.folio.oaipmh.service.impl;
 
+import static org.folio.oaipmh.Constants.OKAPI_URL;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
+import java.sql.Connection;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
 import javax.ws.rs.NotFoundException;
 
+import org.folio.liquibase.LiquibaseUtil;
+import org.folio.liquibase.SingleConnectionProvider;
+import org.folio.oaipmh.dao.PostgresClientFactory;
 import org.folio.oaipmh.dao.SetDao;
 import org.folio.oaipmh.dao.impl.SetDaoImpl;
 import org.folio.oaipmh.service.SetService;
+import org.folio.rest.impl.OkapiMockServer;
 import org.folio.rest.jaxrs.model.Set;
+import org.folio.rest.jaxrs.model.SetsFilteringCondition;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.tools.utils.NetworkUtils;
+import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+import io.vertx.core.Vertx;
 import io.vertx.junit5.VertxExtension;
 import io.vertx.junit5.VertxTestContext;
 
 @ExtendWith(VertxExtension.class)
-class SetServiceImplTest extends AbstractServiceTest {
+class SetServiceImplTest {
 
+  private static final String TEST_TENANT_ID = "oaiTest";
   private static final String EXISTENT_SET_ID = "16287799-d37a-49fb-ac8c-09e9e9fcbd4d";
   private static final String NONEXISTENT_SET_ID = "a3bd69dd-d50b-4aa6-accb-c1f9abaada55";
   private static final String TEST_USER_ID = "30fde4be-2d1a-4546-8d6c-b468caca2720";
 
+  private static final String LOCATION = "location";
+  private static final String ILL_POLICIES = "illPolicy";
```

Comment:
See my comments above about having to define these more than once.

Reply:
I've moved it to Constants.java